src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/LeoRoutes.scala

```diff
@@ -36,11 +36,13 @@ class LeoRoutes(val leonardoService: LeonardoService, val proxyService: ProxySer
 
   def leoRoutes: Route =
     path("cluster" / Segment / Segment) { (googleProject, clusterName) =>
-      put {
-        entity(as[ClusterRequest]) { cluster =>
-          complete {
-            leonardoService.createCluster(GoogleProject(googleProject), ClusterName(clusterName), cluster).map { cluster =>
-              StatusCodes.OK -> cluster
+      requireUserInfo { userInfo =>
```

Comment:
I added `requireUserInfo` to all `/api/` routes. The `userInfo` variable is not used anywhere yet (but will be soon!)